### PR TITLE
Added back selected fields sidebar

### PR DESCRIPTION
**Pull request by davish**
Added back selected fields sidebar for the demo tomorrow

image

### CODE REVIEW

File: frontend/pages/reports.js [hunk 1]
```diff
@@ -99,130 +122,149 @@ const Reports = ({ nameToCode }) => {
...
+        <div className="box">
+          <h3 className="title is-4" style={{ color: CLUBS_GREY }}>
+            Select fields to include
+          </h3>
+          <div>
+            <Flex>
+              {Object.keys(fields).map(group =>
+                generateCheckboxGroup(group, fields[group])
+              )}
+            </Flex>
```
Comment (pawalt) [inline frontend/pages/reports.js]: 💪💪💪

File: frontend/pages/reports.js [hunk 2]
```diff
@@ -28,6 +28,29 @@ const serializeParams = params => {
...
 }
 
+const HoverListElement = s.li`
+  span {
+    display: none;
+  }
+  &:hover span {
+    display: inline;
+    cursor: pointer;
+  }
```
Comment (cacabo) [inline frontend/pages/reports.js]: This doesn't really work with mobile—do we care about that use case?

Also could we maybe try like `margin-left: auto` so all the icons will be aligned on the right side of the sidebar? Might be better UI

**Comment by ezwang** [comment]
@davish does this work on smaller screens? I left it out because of that, but we can add it back temporarily and fix the mobile issue after the demo!

**Comment by davish** [comment]
Yeah, I don't think this will work on mobile. But I also don't see OSA generating reports on mobile, I think it's fine to not support mobile for this view, since you need to download the excel file.

**Comment by ezwang** [comment]
@davish maybe a rule to just hide the sidebar if we're on mobile? That way the page is at least still functional. The ideal fix would just be to prevent the overlap by shifting the content of the page downwards.

**Pull request merged**